packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -1,28 +1,32 @@
 import { Type } from '@openmrs/esm-framework';
 
 export const configSchema = {
-  concepts: {
+  resultsViewerConcepts: {
     _type: Type.Array,
     _elements: {
       conceptUuid: {
         _type: Type.UUID,
-        _description: 'UUID of concept to load from /obstree',
+        _description: `UUID of a test or a concept set containing tests as members, members' members, and so on. Test results will be loaded by querying the REST /obstree endpoint with this concept.`,
       },
       defaultOpen: {
         _type: Type.Boolean,
-        _description: 'Set default behavior of filter accordion',
+        _description:
+          'Each concept set displays the test results it contains in an accordion. Should the accordion be open by default?',
```

Comment:
Nit: Is this meant as an actual question or is it the meant more as "this question that the provided property value answers"?

Reply:
It's the question that the provided property value answers. Right, I guess it is more intuitive to use phrasing like "Set to `true` to make the accordion open by default."